dev/bots/suite_runners/run_web_tests.dart

```diff
@@ -153,48 +147,58 @@ class WebTestsSuite {
       ],
 
       // This test doesn't do anything interesting w.r.t. rendering, so we don't run the full build mode x renderer matrix.
-      () => _runWebE2eTest('profile_diagnostics_integration', buildMode: 'debug', renderer: 'html'),
+      () => _runWebE2eTest(
+        'profile_diagnostics_integration',
+        buildMode: 'debug',
+        renderer: 'canvaskit',
+      ),
       () => _runWebE2eTest(
         'profile_diagnostics_integration',
         buildMode: 'profile',
         renderer: 'canvaskit',
       ),
-      () =>
-          _runWebE2eTest('profile_diagnostics_integration', buildMode: 'release', renderer: 'html'),
+      () => _runWebE2eTest(
+        'profile_diagnostics_integration',
+        buildMode: 'release',
+        renderer: 'canvaskit',
+      ),
 
       // This test is only known to work in debug mode.
-      () => _runWebE2eTest('scroll_wheel_integration', buildMode: 'debug', renderer: 'html'),
+      () => _runWebE2eTest('scroll_wheel_integration', buildMode: 'debug', renderer: 'canvaskit'),
 
       // This test doesn't do anything interesting w.r.t. rendering, so we don't run the full build mode x renderer matrix.
       // These tests have been extremely flaky, so we are temporarily disabling them until we figure out how to make them more robust.
       () => _runWebE2eTest('text_editing_integration', buildMode: 'debug', renderer: 'canvaskit'),
-      () => _runWebE2eTest('text_editing_integration', buildMode: 'profile', renderer: 'html'),
-      () => _runWebE2eTest('text_editing_integration', buildMode: 'release', renderer: 'html'),
+      () => _runWebE2eTest('text_editing_integration', buildMode: 'profile', renderer: 'canvaskit'),
+      () => _runWebE2eTest('text_editing_integration', buildMode: 'release', renderer: 'canvaskit'),
 
       // This test doesn't do anything interesting w.r.t. rendering, so we don't run the full build mode x renderer matrix.
-      () => _runWebE2eTest('url_strategy_integration', buildMode: 'debug', renderer: 'html'),
+      () => _runWebE2eTest('url_strategy_integration', buildMode: 'debug', renderer: 'canvaskit'),
       () => _runWebE2eTest('url_strategy_integration', buildMode: 'profile', renderer: 'canvaskit'),
-      () => _runWebE2eTest('url_strategy_integration', buildMode: 'release', renderer: 'html'),
+      () => _runWebE2eTest('url_strategy_integration', buildMode: 'release', renderer: 'canvaskit'),
 
       // This test doesn't do anything interesting w.r.t. rendering, so we don't run the full build mode x renderer matrix.
       () => _runWebE2eTest(
         'capabilities_integration_canvaskit',
         buildMode: 'debug',
-        renderer: 'auto',
+        renderer: 'canvaskit',
       ),
       () => _runWebE2eTest(
         'capabilities_integration_canvaskit',
         buildMode: 'profile',
         renderer: 'canvaskit',
       ),
-      () => _runWebE2eTest('capabilities_integration_html', buildMode: 'release', renderer: 'html'),
```

Comment:
Will this still work? It seems like you deleted this test

Reply:
Oh, nice catch! I wonder why this didn't fail in CI. I'll have to check this.